azurerm/config.go

```diff
@@ -323,22 +324,15 @@ func getArmClient(c *authentication.Config) (*ArmClient, error) {
 		return keyVaultSpt, nil
 	})
 
-	csc := containerservice.NewContainerServicesClientWithBaseURI(endpoint, c.SubscriptionID)
```

Comment:
I'm at a loss as to why this was removed as it seems unrelated?

Reply:
So this is actually being registered twice in master at the moment (due to a previous merge conflict):

https://github.com/terraform-providers/terraform-provider-azurerm/blob/f76c05bc2cc04e0a1974d0182497abc07abeba18/azurerm/config.go#L326

https://github.com/terraform-providers/terraform-provider-azurerm/blob/f76c05bc2cc04e0a1974d0182497abc07abeba18/azurerm/config.go#L503

this consolidates this into one